radio/src/gui/horus/view_main.cpp

```diff
@@ -29,6 +29,8 @@
 #define TRIM_LEN                       80
 #define POTS_LINE_Y                    (LCD_H-20)
 
+extern uint8_t lastMonitorPage;
```

Comment:
It should be inside menus.h then, I think there is another location it could be removed